### ISSUE TITLE
Add Scottish Environmental Protection Agency (SEPA) as source

### ISSUE THREAD

**Issue by KarenJewell** [OpenDataScotland/the_od_bods#181]
**Is your feature request related to a problem? Please describe.**
This one is HUGE. SEPA publish lots of data but not in a portal so this one will need to be broken into parts to be tackled. My recommendation is to create a new issue for the sub-section to work on and link back here.
Note also the SEPA may be moving portals so check in with them before starting work.
https://www.sepa.org.uk/environment/environmental-data/

**Comment by KarenJewell** [comment]
Have sent an email to SEPA asking them if they have plans to move to a data portal.

**Comment by fozy81** [comment]
Suggest starting with adding this arcgis endpoint https://map.sepa.org.uk/server/rest/services/Open  (OGL). This is unlikely to change and lots of low hanging fruit. Happy to assign this to me.
